task/05-regex-tasks.js

```diff
@@ -31,7 +31,9 @@
  * @return {RegExp}
  */
 function getRegexForGuid() {
-   throw new Error('Not implemented');
+   return /{[\d, a-f, A-F]{8}-[\d, a-f, A-F]{4}-[\d, a-f, A-F]{4}-[\d, a-f, A-F]{4}-[\d, a-f, A-F]{12}}/;
+
+   //throw new Error('Not implemented');
 }
```

Comment:
three times repeating part [\d, a-f, A-F]{4}- could be reduced to ([\d, a-f, A-F]{4}-){3}
unnecessary `,` and ` ` signs